src/main/java/com/capitalone/dashboard/event/RelatedCollectorItemEventListener.java

```diff
@@ -30,7 +31,12 @@ public RelatedCollectorItemEventListener(
     public void onAfterSave(AfterSaveEvent<RelatedCollectorItem> event) {
         RelatedCollectorItem relatedCollectorItem = event.getSource();
         try {
-            syncDashboard.sync(relatedCollectorItem);
+            if(relatedCollectorItem.getReason().equalsIgnoreCase(Reason.ARTIFACT_REASON.getAction())){
```

Comment:
this can be simplified as 
 syncDashboard.sync(relatedCollectorItem, !StringUtils.equalsIgnoreCase(Reason.ARTIFACT_REASON.getAction(), (relatedCollectorItem.getReason())